tdcommon/src/main/java/thredds/server/catalog/tracker/DatasetTrackerChronicle.java

```diff
@@ -207,19 +210,29 @@ else if (!path.equals(accessPath)) { // LOOK must put all for restrict
 
     // changed = true;
     DatasetExt dsext = new DatasetExt(catId, dataset.getRestrictAccess(), ncml);
+    if (datasetMap == null) {
+      catalogInitLog.error("Catalog chronicle map not properly initialized. Can't track dataset {}", dataset);
```

Comment:
I'll note that this doesn't print out the name of the dataset, it prints out a repr of the dataset instance. I just followed the same pattern as above, if that's good enough.